src/Database/Query.php

```diff
@@ -1820,7 +1820,7 @@ public function delete(?string $table = null)
      *
      * Epliog content is raw SQL and not suitable for use with user supplied data.
      *
-     * @param string|\Cake\Database\Expression\QueryExpression|null $expression The expression to be appended
+     * @param string|\Cake\Database\Expression\QueryExpression|\Cake\ORM\Query|null $expression The expression to be appended
```

Comment:
```suggestion
     * @param string|\Cake\Database\ExpressionInterface|null $expression The expression to be appended
```

IMHO there's no need to restrict this to concrete implementations, given that `epilog()` is ment for appending arbitrary SQL.

Reply:
I agree.